x/oracle/internal/keeper/params.go

```diff
@@ -41,6 +41,24 @@ func (k Keeper) Whitelist(ctx sdk.Context) (res types.DenomList) {
 	return
 }
 
+// SlashFraction returns oracle voting penalty rate
```

Comment:
Does this need to be stored in the keeper? Could this not just be accessed through the paramkeeper?

Reply:
This is just interface for easy lookup, the params are stored in paramkeeper.